src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcTokenHandler.php

```diff
@@ -93,7 +94,7 @@ public function getUserBadgeFrom(string $accessToken): UserBadge
             }
 
             // UserLoader argument can be overridden by a UserProvider on AccessTokenAuthenticator::authenticate
-            return new UserBadge($claims[$this->claim], fn () => $this->createUser($claims), $claims);
+            return new UserBadge($claims[$this->claim], new FallbackUserLoader(fn () => $this->createUser($claims)), $claims);
```

Comment:
Might need the same fix on [OidcUserInfoTokenHandler](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcUserInfoTokenHandler.php)